src/lib.rs

```diff
@@ -86,6 +116,30 @@ pub fn initialize(_argc: *const isize, _argv: *const *const *const u8) -> isize
 /// # mod my_crate { pub fn parse(_: &[u8]) -> Result<(), ()> { unimplemented!() } }
 /// ```
 ///
+/// ## Rejecting Inputs
+///
+/// To indicate whether an input should be kept in or rejected from the corpus,
+/// return a [Corpus] value from your fuzz target. For example:
```

Comment:
Can you expand a bit here for users and explain *why* they might want to reject an input from the corpus? Basically restating what the libfuzzer docs say should be good.

Reply:
Done!